### PR TITLE
Enable builds on CircleLinux

### PR DESCRIPTION
**Pull request by bella485**

### CODE REVIEW

**Comment by bella485** [comment]
Hello Jijun, 

Circle Linux 在向一些 upstream项目中添加distro support.
按照理解，提交了一个pullrequest，如果有任何问题请联系我。

File: MakeTools/funcs-ext.sh [hunk 1]
```diff
@@ -768,6 +771,9 @@ function os_vendor()
 		elif [ $rval = 'almalinux' ]
 		then
 			rval=redhat
+        elif [ $rval = 'circle' ]
+		then
+			rval=redhat
```
Comment (jijunwan) [inline MakeTools/funcs-ext.sh:776]: Please adjust indentation.

File: MakeTools/funcs-ext.sh [hunk 2]
```diff
@@ -850,6 +856,10 @@ function os_vendor_version()
 		then
 			# AlmaLinux
 			rval="ES"`cat /etc/redhat-release | sed -r 's/^.+([[:digit:]])\.([[:digit:]]).+$/\1\2/'`
+        elif grep -qi circle /etc/redhat-release
+		then
+			# Circle Linux 
+			rval="ES"`cat /etc/redhat-release | sed -r 's/^.+([[:digit:]])\.([[:digit:]]).+$/\1\2/'`
```
Comment (jijunwan) [inline MakeTools/funcs-ext.sh:862]: Please adjust indentation.

**Review by jijunwan — COMMENTED** [review]
Thanks for your contribution. It looks good to me, except couple minor indentation issues.

Also suggest change your Pull Request title to "Enable builds on CircleLinux" as we do not officially support it yet, and no full validation on it for now.

Also please confirm you have followed README to test building fastfabric in Circle Linux. We will do a quick test on our side to ensure no regression issue as well.

**Comment by bella485** [comment]
OK,  some type mistake fixed, thanks a lot.

**Comment by bella485** [comment]
Yes,I have followed README to test building fastfabric on Circle Linux, rpm packages were created.

Thanks alot your review.

Regards.

**Pull request merged**